FindTown/FindTownUI/Sources/FindTownUI/DropDown/DropDown.swift

```diff
@@ -0,0 +1,224 @@
+//
+//  File.swift
+//  
+//
+//  Created by 장선영 on 2022/12/29.
+//
+
+import Foundation
+import UIKit
+
+public enum DropDownStatus {
+    case selected
+    case nonSelected
+}
+
+/// 커스텀 드롭다운, 드롭다운의 세로 길이가 데이터 목록을 보여주는 tableView의 세로 길이가 됨
+public class DropDown: UIView {
+    
+    /// 0번째에는 데이터가 아닌 안내 텍스트 (ex:자치구(선택)) 가 들어감
+    var dataSource = ["test1", "Test2","test3","Test4","test5"]
+    
```

Comment:
```suggestion
    /// 0번째에는 데이터가 아닌 안내 텍스트 (ex:자치구(선택)) 가 들어감
    var dataSource: [String]
    
```

dataSource를 받으니까 init할 때 받으니까 이 부분은 이렇게 해도 괜찮지 않을까요..?!
아마 지우시는걸 깜빡하신거 같아요!! 😀

Reply:
앗..!!! ㅋㅋㅋㅋ 수정하겠습니다 감사해요 😅